lib/config/theme/theme_palette/default_palette.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+
+class DefaultPalette {
+  static const MaterialColor kToDark = MaterialColor(
```

Comment:
you dont need it, and you won't need in the future, bcs your app has only light design